api/tests/routes/backoffice/offers_test.py

```diff
@@ -1136,6 +1137,29 @@ def test_validate_offer(self, legit_user, authenticated_client):
 
         assert offer_to_validate.isActive is True
         assert offer_to_validate.lastValidationType == OfferValidationType.MANUAL
+        assert offer_to_validate.lastValidationPrice == decimal.Decimal("10.1")
+
+    def test_validate_offer_without_stocks(self, legit_user, authenticated_client):
+        offer_to_validate = offers_factories.OfferFactory(validation=offers_models.OfferValidationStatus.REJECTED)
+
+        response = self.post_to_endpoint(authenticated_client, offer_id=offer_to_validate.id)
+        assert response.status_code == 303
+
+        expected_url = url_for("backoffice_web.offer.list_offers", _external=True)
+        assert response.location == expected_url
+
+        offer_list_url = url_for("backoffice_web.offer.list_offers", q=offer_to_validate.id, _external=True)
+        response = authenticated_client.get(offer_list_url)
+
+        assert response.status_code == 200
+        row = html_parser.extract_table_rows(response.data)
+        assert len(row) == 1
+        assert row[0]["État"] == "Validée"
+        assert row[0]["Dernière validation"] == (datetime.date.today()).strftime("%d/%m/%Y")
```

Comment:
A priori pas besoin de tester à nouveau le rendu de la liste des offres, l'action est de valider, c'est testé en-dessous
```suggestion
        expected_url = url_for("backoffice_web.offer.list_offers", _external=True)
        assert response.location == expected_url

        offer_list_url = url_for("backoffice_web.offer.list_offers", q=offer_to_validate.id, _external=True)
        response = authenticated_client.get(offer_list_url)

        assert response.status_code == 200
        row = html_parser.extract_table_rows(response.data)
        assert len(row) == 1
        assert row[0]["État"] == "Validée"
        assert row[0]["Dernière validation"] == (datetime.date.today()).strftime("%d/%m/%Y")
```